[MRG] Raise error in OneHotEncoder.inverse_transform (#14982)
scikit-learn/scikit-learn: doc/whats_new/v0.24.rst

  :class:`preprocessing.KBinsDiscretizer`.
  :pr:`16335` by :user:`Arthur Imbert <Henley13>`.

> - |Fix| Raise error on :meth:`sklearn.preprocessing.OneHotEncoder.inverse_transform` on input cases
>   when it is expected in the case of ``handle_uknown='error'``.
>   :pr:`14982` by :user:`Kevin Winata <kwinata>`.

Reviewer: ```suggestion
- |Fix| Raise error on 
  :meth:`sklearn.preprocessing.OneHotEncoder.inverse_transform`
  when `handle_unknown='error'` and `drop=None` for samples
  encoded as all zeros. :pr:`14982` by 
  :user:`Kevin Winata <kwinata>`.
```
